test/utils.js

```diff
@@ -20,9 +20,37 @@
 const GLib = imports.gi.GLib;
 const Gio = imports.gi.Gio;
 
+const EosShard = imports.gi.EosShard;
+
 function getTestFile(fn) {
     let datadir = GLib.getenv('G_TEST_SRCDIR');
     if (!datadir)
         datadir = '';
     return Gio.File.new_for_path(GLib.build_filenamev([datadir, 'test/data', fn]));
 }
+
+function writeTestShard(shard_path) {
```

Comment:
I actually prefer to repeat myself in tests rather than deduplicate, because it makes tests easier to read and debug IMO.